kivy/_event.pyx

```diff
@@ -228,7 +228,16 @@ cdef class EventDispatcher(ObjectWithUid):
         prop_args = {
             k: kwargs.pop(k) for k in list(kwargs.keys()) if k in properties}
         self._kwargs_applied_init = set(prop_args.keys()) if prop_args else set()
-        super(EventDispatcher, self).__init__(**kwargs)
+
+        try: # at this point any kwargs passed may go to object which will error and cause confusion
+           super(EventDispatcher, self).__init__(**kwargs)
+        except TypeError as e:
+           if kwargs and "object.__init__() takes exactly one argument" in e:
+               raise ValueError(
+                  'Unexpected properties {}, valid properties are {}'.format(
```

Comment:
```suggestion
                  'Properties "{}" past to __init__ may not be valid '
                  'property names. Valid properties are "{}"'.format(
```

Reply:
Please use the message I suggested because it's clearer and hedges in case the exception is because of an different error, just in case.